EFCore2Playgrounds/Program.cs

```diff
@@ -34,10 +34,8 @@ public class Order
     {
         public int Id { get; set; }
         private int StateId;
-        private State _state;
-        private State State => _state;
```

Comment:
I've never seen this pattern before, whats the advantage of this?

Reply:
OK I see this is a read-only property with _state as the backing field

Reply:
The idea was to hidden the navigation property to the consumer (Encapsulation) and support access  through the entity.